esp/bindings/SOAP/Platform/soapbind.cpp

```diff
@@ -271,24 +271,37 @@ int CHttpSoapBinding::HandleSoapRequest(CHttpRequest* request, CHttpResponse* re
     return 0;
 }
 
-
-static IPropertyTree *createSecClientConfig(const char *clientCertPath, const char *clientPrivateKey, const char *caCertsPath, bool acceptSelfSigned)
+static IPropertyTree *createSecClientConfig(const char *clientCertFileorBuf, const char *clientPrivKeyFileorBuf, const char *caCertsPathorBuf, bool acceptSelfSigned)
 {
     Owned<IPropertyTree> info = createPTree();
 
-    if (!isEmptyString(clientCertPath))
+    if (!isEmptyString(clientCertFileorBuf))
     {
-        info->setProp("certificate", clientCertPath);
-        if (!isEmptyString(clientPrivateKey))
-            info->setProp("privatekey", clientPrivateKey);
+        if (isPEMBuffer(clientCertFileorBuf))
```

Comment:
Can this always set "certificate" and then have the PEMbuffer test later?  A later clean up PR would be fine.

Reply:
Yes, but there is also the pem / \@path difference, do we leave that in or also remove ?